p4dlog.go

```diff
@@ -737,7 +737,8 @@ var trackReplicaPull = "--- replica/pull"
 var trackStorage = "--- storageup/"
 var reCmdTrigger = regexp.MustCompile(` trigger ([^ ]+)$`)
 var reTriggerLapse = regexp.MustCompile(`^lapse (\d+)s`)
-var reTriggerLapse2 = regexp.MustCompile(`^lapse \.(\d+)s`)
+var reTriggerLapse2 = regexp.MustCompile(`lapse \.(\d+)s`)
+var reTriggerLapse3 = regexp.MustCompile(`lapse [0-9]\.(\d+)s`)
 var prefixTrackRPC = "--- rpc msgs/size in+out "
```

Comment:
Hi John, Iine 741 is not correct because you need to include the seconds inside the match group.
I also think we could combine the various regexes into something like:

var reTriggerLapse = regexp.MustCompile(`^lapse ([.0-9]+)s`)

And then we need to parse the resulting match group as a float. Just need to test that the regex will match things like:

.123s
1.123s
33s
34.123s